### PR TITLE
Miscellaneous refactors

### PR DESCRIPTION
**Pull request by gshilin**
* Use hook `useTranslation` instead of HOC `withTranslation`
* File should have the same name as a component inside it: rename list.js -> SimpleModeList.js
* Added `isToday` helper
* Added `refetch` functionality to `SimpleModeList`
* Supply proper parameters to `WipErr`
* Other minor changes

### CODE REVIEW

**Review by davgur — APPROVED** [review]

File: src/helpers/utils.js [hunk 1]
```diff
@@ -377,8 +377,9 @@ export const getSectionForTranslation = content_type => {
   }
 };
 
-export const noop = () => {
-};
+export const isToday = selectedDate => moment().isSame(moment(selectedDate), 'date');
```
Comment (bbfsdev) [inline src/helpers/utils.js:380]: Looks like premature refactoring. I would keep this in the ONE place this is being used.
Too early for utils.

File: src/components/Sections/SimpleMode/SimpleModeList.js [hunk 2]
```diff
@@ -39,11 +43,10 @@ const SimpleModeList = ({ filesLanguages, t, renderUnit, selectedDate }) => {
     return wipErr;
   }
 
-  if (!isSuccess || (lessons.length === 0 && others.length === 0)) {
```
Comment (bbfsdev) [inline src/components/Sections/SimpleMode/SimpleModeList.js:42]: Why did you remove this condition?

File: src/components/Sections/SimpleMode/SimpleModeList.js [hunk 3]
```diff
@@ -1,36 +1,40 @@
...
 
-  const { isError, isLoading, isSuccess, error, data } = useSimpleModeQuery({
+  const { isError, isLoading, isSuccess, error, data, refetch } = useSimpleModeQuery({
     date            : selectedDate,
     uiLanguage,
     contentLanguages: filesLanguages
   });
 
+  const callback = useCallback(() => isToday(selectedDate) ? refetch : noop, [selectedDate, refetch]);
+  useInterval(callback, 60 * 1000);
```
Comment (bbfsdev) [inline src/components/Sections/SimpleMode/SimpleModeList.js:30]: What that fixes?
Why is that needed?

File: src/helpers/utils.js (hunk 1 above)
Comment (gshilin) [reply src/helpers/utils.js:380]: it looks much more related to utils and not to any code.

File: src/components/Sections/SimpleMode/SimpleModeList.js (hunk 2 above)
Comment (gshilin) [reply src/components/Sections/SimpleMode/SimpleModeList.js:42]: reverted

File: src/components/Sections/SimpleMode/SimpleModeList.js (hunk 3 above)
Comment (gshilin) [reply src/components/Sections/SimpleMode/SimpleModeList.js:30]: Imagine: you're looking on a download page. Then some part of lesson is over. Now you should F5 until it will be available to download. This code eliminates it by trying to refetch data every minute

File: src/helpers/utils.js (hunk 1 above)
Comment (bbfsdev) [reply src/helpers/utils.js:380]: Don't have a strong opinion about his. Though it will be easier for readers inline.

File: src/components/Sections/SimpleMode/SimpleModeList.js (hunk 3 above)
Comment (bbfsdev) [reply src/components/Sections/SimpleMode/SimpleModeList.js:30]: Nice addition.
